### PR TITLE
Add support Climax WLS-15ZBS Water Leakage Sensor

### PR DESCRIPTION
**Pull request by patriciocl**

### CODE REVIEW

File: devices.js [hunk 1]
```diff
@@ -8662,15 +8662,18 @@ const devices = [
...
+
+    },
+    {
+        zigbeeModel: ['WS15_00.00.00.10TC'],
+        model: 'WLS-15ZBS',
+        vendor: 'Climax',
+        description: 'Water leakage sensor',
+        fromZigbee: [fz.ias_water_leak_alarm_1, fz.battery],
         toZigbee: [],
-        meta: {configureKey: 1},
```
Comment (Koenkk) [inline devices.js:8666]: Why was this removed?

Comment (patriciocl) [reply devices.js:8666]: check it really is not necessary it is the same device as zigbeeModel: ['902010/24'], maybe it is better to group devices

Comment (Koenkk) [reply devices.js:8666]: But did you have the device to test it? (and confirmed it was still working correctly after repairing)

Comment (patriciocl) [reply devices.js:8666]: Work fine after repairing . now work siren

```
info  2021-02-05 13:26:21: Accepting joining not in blocklist device '0x00124b00168change'
info  2021-02-05 13:26:21: Device '0x00124b00168change' joined
info  2021-02-05 13:26:21: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0x00124b00168change","ieee_address":"0x00124b00168change"},"type":"device_joined"}'
info  2021-02-05 13:26:21: Starting interview of '0x00124b00168change'
info  2021-02-05 13:26:21: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0x00124b00168change","ieee_address":"0x00124b00168change","status":"started"},"type":"device_interview"}'
info  2021-02-05 13:26:21: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":{"friendly_name":"0x00124b00168change"},"type":"device_connected"}'
info  2021-02-05 13:26:21: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"interview_started","meta":{"friendly_name":"0x00124b00168change"},"type":"pairing"}'
info  2021-02-05 13:26:21: MQTT publish: topic 'zigbee2mqtt/0x00124b00168change/availability', payload 'online'
info  2021-02-05 13:26:21: Accepting joining not in blocklist device '0x00124b00168change'
debug 2021-02-05 13:26:22: Device '0x00124b00168change' announced itself
info  2021-02-05 13:26:22: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0x00124b00168change","ieee_address":"0x00124b00168change"},"type":"device_announce"}'
info  2021-02-05 13:26:22: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"announce","meta":{"friendly_name":"0x00124b00168change"},"type":"device_announced"}'
debug 2021-02-05 13:26:23: Received Zigbee message from '0x00124b00168change', type 'readResponse', cluster 'genBasic', data '{"modelId":"SD8SC_00.00.03.12TC"}' from endpoint 1 with groupID 0
debug 2021-02-05 13:26:23: No converter available for 'SD-8SCZBS' with cluster 'genBasic' and type 'readResponse' and data '{"modelId":"SD8SC_00.00.03.12TC"}'
debug 2021-02-05 13:26:24: Received Zigbee message from '0x00124b00168change', type 'readResponse', cluster 'genBasic', data '{"manufacturerName":"ClimaxTechnology"}' from endpoint 1 with groupID 0
debug 2021-02-05 13:26:24: No converter available for 'SD-8SCZBS' with cluster 'genBasic' and type 'readResponse' and data '{"manufacturerName":"ClimaxTechnology"}'
debug 2021-02-05 13:26:24: Received Zigbee message from '0x00124b00168change', type 'readResponse', cluster 'genBasic', data '{"powerSource":3}' from endpoint 1 with groupID 0
debug 2021-02-05 13:26:24: No converter available for 'SD-8SCZBS' with cluster 'genBasic' and type 'readResponse' and data '{"powerSource":3}'
debug 2021-02-05 13:26:24: Received Zigbee message from '0x00124b00168change', type 'readResponse', cluster 'genBasic', data '{"zclVersion":1}' from endpoint 1 with groupID 0
debug 2021-02-05 13:26:24: No converter available for 'SD-8SCZBS' with cluster 'genBasic' and type 'readResponse' and data '{"zclVersion":1}'
debug 2021-02-05 13:26:24: Received Zigbee message from '0x00124b00168change', type 'readResponse', cluster 'genBasic', data '{"appVersion":20}' from endpoint 1 with groupID 0
debug 2021-02-05 13:26:24: No converter available for 'SD-8SCZBS' with cluster 'genBasic' and type 'readResponse' and data '{"appVersion":20}'
debug 2021-02-05 13:26:24: Received Zigbee message from '0x00124b00168change', type 'readResponse', cluster 'genBasic', data '{}' from endpoint 1 with groupID 0
debug 2021-02-05 13:26:24: No converter available for 'SD-8SCZBS' with cluster 'genBasic' and type 'readResponse' and data '{}'
debug 2021-02-05 13:26:24: Received Zigbee message from '0x00124b00168change', type 'readResponse', cluster 'genBasic', data '{"hwVersion":0}' from endpoint 1 with groupID 0
debug 2021-02-05 13:26:24: No converter available for 'SD-8SCZBS' with cluster 'genBasic' and type 'readResponse' and data '{"hwVersion":0}'
debug 2021-02-05 13:26:25: Received Zigbee message from '0x00124b00168change', type 'readResponse', cluster 'genBasic', data '{"dateCode":"20161115        "}' from endpoint 1 with groupID 0
debug 2021-02-05 13:26:25: No converter available for 'SD-8SCZBS' with cluster 'genBasic' and type 'readResponse' and data '{"dateCode":"20161115        "}'
debug 2021-02-05 13:26:25: Received Zigbee message from '0x00124b00168change', type 'readResponse', cluster 'genBasic', data '{}' from endpoint 1 with groupID 0
debug 2021-02-05 13:26:25: No converter available for 'SD-8SCZBS' with cluster 'genBasic' and type 'readResponse' and data '{}'
debug 2021-02-05 13:26:25: Received Zigbee message from '0x00124b00168change', type 'readResponse', cluster 'ssIasZone', data '{"iasCieAddr":"0x0000000000000000","zoneState":1}' from endpoint 1 with groupID 0
debug 2021-02-05 13:26:25: No converter available for 'SD-8SCZBS' with cluster 'ssIasZone' and type 'readResponse' and data '{"iasCieAddr":"0x0000000000000000","zoneState":1}'
debug 2021-02-05 13:26:25: Received Zigbee message from '0x00158d0001d2c4ab', type 'readResponse', cluster 'genBasic', data '{"zclVersion":1}' from endpoint 1 with groupID 0
debug 2021-02-05 13:26:25: Successfully pinged '0x00158d0001d2c4ab'
debug 2021-02-05 13:26:26: Received Zigbee message from 'bot-pato', type 'attributeReport', cluster 'genBasic', data '{"65282":[{"elmType":16,"elmVal":1},{"elmType":33,"elmVal":3022},{"elmType":33,"elmVal":17320},{"elmType":36,"elmVal":[0,1]},{"elmType":33,"elmVal":328},{"elmType":32,"elmVal":91}]}' from endpoint 1 with groupID 0
info  2021-02-05 13:26:26: MQTT publish: topic 'zigbee2mqtt/bot-pato', payload '{"battery":100,"last_seen":"2021-02-05T13:26:26.996Z","linkquality":99,"voltage":3022}'
debug 2021-02-05 13:26:27: Received Zigbee message from '0x00124b00168change', type 'readResponse', cluster 'ssIasZone', data '{"iasCieAddr":"0x00124b000be87115","zoneState":1}' from endpoint 1 with groupID 0
debug 2021-02-05 13:26:27: No converter available for 'SD-8SCZBS' with cluster 'ssIasZone' and type 'readResponse' and data '{"iasCieAddr":"0x00124b000be87115","zoneState":1}'
info  2021-02-05 13:26:27: Successfully interviewed '0x00124b00168change', device has successfully been paired
info  2021-02-05 13:26:27: Device '0x00124b00168change' is supported, identified as: Climax Optical smoke detector (hardware version v1) (SD-8SCZBS)
info  2021-02-05 13:26:27: MQTT publish: topic 'homeassistant/binary_sensor/0x00124b00168change/smoke/config', payload '{"availability":[{"topic":"zigbee2mqtt/bridge/state"},{"topic":"zigbee2mqtt/0x00124b00168change/availability"}],"device":{"identifiers":["zigbee2mqtt_0x00124b00168change"],"manufacturer":"Climax","model":"Optical smoke detector (hardware version v1) (SD-8SCZBS)","name":"0x00124b00168change","sw_version":"Zigbee2MQTT 1.17.1-dev"},"device_class":"smoke","json_attributes_topic":"zigbee2mqtt/0x00124b00168change","name":"0x00124b00168change smoke","payload_off":false,"payload_on":true,"state_topic":"zigbee2mqtt/0x00124b00168change","unique_id":"0x00124b00168change_smoke_zigbee2mqtt","value_template":"{{ value_json.smoke }}"}'
info  2021-02-05 13:26:27: MQTT publish: topic 'homeassistant/sensor/0x00124b00168change/battery/config', payload '{"availability":[{"topic":"zigbee2mqtt/bridge/state"},{"topic":"zigbee2mqtt/0x00124b00168change/availability"}],"device":{"identifiers":["zigbee2mqtt_0x00124b00168change"],"manufacturer":"Climax","model":"Optical smoke detector (hardware version v1) (SD-8SCZBS)","name":"0x00124b00168change","sw_version":"Zigbee2MQTT 1.17.1-dev"},"device_class":"battery","json_attributes_topic":"zigbee2mqtt/0x00124b00168change","name":"0x00124b00168change battery","state_topic":"zigbee2mqtt/0x00124b00168change","unique_id":"0x00124b00168change_battery_zigbee2mqtt","unit_of_measurement":"%","value_template":"{{ value_json.battery }}"}'
info  2021-02-05 13:26:27: MQTT publish: topic 'homeassistant/binary_sensor/0x00124b00168change/battery_low/config', payload '{"availability":[{"topic":"zigbee2mqtt/bridge/state"},{"topic":"zigbee2mqtt/0x00124b00168change/availability"}],"device":{"identifiers":["zigbee2mqtt_0x00124b00168change"],"manufacturer":"Climax","model":"Optical smoke detector (hardware version v1) (SD-8SCZBS)","name":"0x00124b00168change","sw_version":"Zigbee2MQTT 1.17.1-dev"},"device_class":"battery","json_attributes_topic":"zigbee2mqtt/0x00124b00168change","name":"0x00124b00168change battery low","payload_off":false,"payload_on":true,"state_topic":"zigbee2mqtt/0x00124b00168change","unique_id":"0x00124b00168change_battery_low_zigbee2mqtt","value_template":"{{ value_json.battery_low }}"}'
info  2021-02-05 13:26:27: MQTT publish: topic 'homeassistant/binary_sensor/0x00124b00168change/tamper/config', payload '{"availability":[{"topic":"zigbee2mqtt/bridge/state"},{"topic":"zigbee2mqtt/0x00124b00168change/availability"}],"device":{"identifiers":["zigbee2mqtt_0x00124b00168change"],"manufacturer":"Climax","model":"Optical smoke detector (hardware version v1) (SD-8SCZBS)","name":"0x00124b00168change","sw_version":"Zigbee2MQTT 1.17.1-dev"},"json_attributes_topic":"zigbee2mqtt/0x00124b00168change","name":"0x00124b00168change tamper","payload_off":false,"payload_on":true,"state_topic":"zigbee2mqtt/0x00124b00168change","unique_id":"0x00124b00168change_tamper_zigbee2mqtt","value_template":"{{ value_json.tamper }}"}'
info  2021-02-05 13:26:27: MQTT publish: topic 'homeassistant/sensor/0x00124b00168change/linkquality/config', payload '{"availability":[{"topic":"zigbee2mqtt/bridge/state"},{"topic":"zigbee2mqtt/0x00124b00168change/availability"}],"device":{"identifiers":["zigbee2mqtt_0x00124b00168change"],"manufacturer":"Climax","model":"Optical smoke detector (hardware version v1) (SD-8SCZBS)","name":"0x00124b00168change","sw_version":"Zigbee2MQTT 1.17.1-dev"},"icon":"mdi:signal","json_attributes_topic":"zigbee2mqtt/0x00124b00168change","name":"0x00124b00168change linkquality","state_topic":"zigbee2mqtt/0x00124b00168change","unique_id":"0x00124b00168change_linkquality_zigbee2mqtt","unit_of_measurement":"lqi","value_template":"{{ value_json.linkquality }}"}'
info  2021-02-05 13:26:27: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"interview_successful","meta":{"description":"Optical smoke detector (hardware version v1)","friendly_name":"0x00124b00168change","model":"SD-8SCZBS","supported":true,"vendor":"Climax"},"type":"pairing"}'
debug 2021-02-05 13:26:27: Received MQTT message on 'homeassistant/binary_sensor/0x00124b00168change/smoke/config' with data '{"availability":[{"topic":"zigbee2mqtt/bridge/state"},{"topic":"zigbee2mqtt/0x00124b00168change/availability"}],"device":{"identifiers":["zigbee2mqtt_0x00124b00168change"],"manufacturer":"Climax","model":"Optical smoke detector (hardware version v1) (SD-8SCZBS)","name":"0x00124b00168change","sw_version":"Zigbee2MQTT 1.17.1-dev"},"device_class":"smoke","json_attributes_topic":"zigbee2mqtt/0x00124b00168change","name":"0x00124b00168change smoke","payload_off":false,"payload_on":true,"state_topic":"zigbee2mqtt/0x00124b00168change","unique_id":"0x00124b00168change_smoke_zigbee2mqtt","value_template":"{{ value_json.smoke }}"}'
debug 2021-02-05 13:26:27: Received MQTT message on 'homeassistant/sensor/0x00124b00168change/battery/config' with data '{"availability":[{"topic":"zigbee2mqtt/bridge/state"},{"topic":"zigbee2mqtt/0x00124b00168change/availability"}],"device":{"identifiers":["zigbee2mqtt_0x00124b00168change"],"manufacturer":"Climax","model":"Optical smoke detector (hardware version v1) (SD-8SCZBS)","name":"0x00124b00168change","sw_version":"Zigbee2MQTT 1.17.1-dev"},"device_class":"battery","json_attributes_topic":"zigbee2mqtt/0x00124b00168change","name":"0x00124b00168change battery","state_topic":"zigbee2mqtt/0x00124b00168change","unique_id":"0x00124b00168change_battery_zigbee2mqtt","unit_of_measurement":"%","value_template":"{{ value_json.battery }}"}'
debug 2021-02-05 13:26:27: Received MQTT message on 'homeassistant/binary_sensor/0x00124b00168change/battery_low/config' with data '{"availability":[{"topic":"zigbee2mqtt/bridge/state"},{"topic":"zigbee2mqtt/0x00124b00168change/availability"}],"device":{"identifiers":["zigbee2mqtt_0x00124b00168change"],"manufacturer":"Climax","model":"Optical smoke detector (hardware version v1) (SD-8SCZBS)","name":"0x00124b00168change","sw_version":"Zigbee2MQTT 1.17.1-dev"},"device_class":"battery","json_attributes_topic":"zigbee2mqtt/0x00124b00168change","name":"0x00124b00168change battery low","payload_off":false,"payload_on":true,"state_topic":"zigbee2mqtt/0x00124b00168change","unique_id":"0x00124b00168change_battery_low_zigbee2mqtt","value_template":"{{ value_json.battery_low }}"}'
debug 2021-02-05 13:26:27: Received MQTT message on 'homeassistant/binary_sensor/0x00124b00168change/tamper/config' with data '{"availability":[{"topic":"zigbee2mqtt/bridge/state"},{"topic":"zigbee2mqtt/0x00124b00168change/availability"}],"device":{"identifiers":["zigbee2mqtt_0x00124b00168change"],"manufacturer":"Climax","model":"Optical smoke detector (hardware version v1) (SD-8SCZBS)","name":"0x00124b00168change","sw_version":"Zigbee2MQTT 1.17.1-dev"},"json_attributes_topic":"zigbee2mqtt/0x00124b00168change","name":"0x00124b00168change tamper","payload_off":false,"payload_on":true,"state_topic":"zigbee2mqtt/0x00124b00168change","unique_id":"0x00124b00168change_tamper_zigbee2mqtt","value_template":"{{ value_json.tamper }}"}'
debug 2021-02-05 13:26:27: Received MQTT message on 'homeassistant/sensor/0x00124b00168change/linkquality/config' with data '{"availability":[{"topic":"zigbee2mqtt/bridge/state"},{"topic":"zigbee2mqtt/0x00124b00168change/availability"}],"device":{"identifiers":["zigbee2mqtt_0x00124b00168change"],"manufacturer":"Climax","model":"Optical smoke detector (hardware version v1) (SD-8SCZBS)","name":"0x00124b00168change","sw_version":"Zigbee2MQTT 1.17.1-dev"},"icon":"mdi:signal","json_attributes_topic":"zigbee2mqtt/0x00124b00168change","name":"0x00124b00168change linkquality","state_topic":"zigbee2mqtt/0x00124b00168change","unique_id":"0x00124b00168change_linkquality_zigbee2mqtt","unit_of_measurement":"lqi","value_template":"{{ value_json.linkquality }}"}'
info  2021-02-05 13:26:31: MQTT publish: topic 'zigbee2mqtt/pir-mama', payload '{"battery":100,"elapsed":90001,"illuminance":12,"illuminance_lux":12,"last_seen":"2021-02-05T13:25:01.271Z","linkquality":34,"occupancy":false,"temperature":25,"voltage":3065}'
debug 2021-02-05 13:26:34: Received Zigbee message from 'led_comedor', type 'readResponse', cluster 'genBasic', data '{"zclVersion":1}' from endpoint 1 with groupID 0
```
